src/components/static/pages/AboutUs.tsx

```diff
@@ -2,116 +2,257 @@ import * as React from "react";
+                role: "Manager of Communication/Branding/Media",
+                quote: "-\"Everybody is a genius. But if you judge a fish by its ability to climb a tree, it will live its whole life believing that it is stupid.\" - Albert Einstein",
+                bio: "Aaron is the most charismatic team member with broad experience in data privacy. He always ensures that every student is having a great time while learning new computer science concepts. Aaron works on reaching out to potential partners at schools and other nonprofit organizations",
+                img: AaronReal,
+                cartoon: Aaron
+            }
+        ];
+
+        return Team.map(person => {
+            this.flag = !this.flag;
+            if (this.flag) {
+                return <Row>
+                    <Col xs={6} md={4}>
+                        <BioPic>
+                            <img src={person.cartoon} alt="team member pic" onMouseOver={e => (e.currentTarget.src =
+                                person.img)} onMouseOut={e => (e.currentTarget.src = person.cartoon)}/>
+                        </BioPic>
+                    </Col>
+                    <Col xs={12} md={8}>
+                        <Bio1> 
+                            <h2>{person.name}</h2>
+                            <h3>{person.role}</h3>
+                            <i><h5>{person.quote}</h5></i>
+                            <br/>
+                            <p>{person.bio}</p>
+                        </Bio1>
+                    </Col>
+                </Row>;
+            }
+
+            return <Row>
+                <Col xs={12} md={8}>
+                    <Bio>
+                        <h2>{person.name}</h2>
+                        <h3>{person.role}</h3>
+                        <i><h5>{person.quote}</h5></i>
+                        <br/>
+                        <p>{person.bio}</p>
+                    </Bio>
+                </Col>
+               <Col xs={6} md={4}>
+                   <BioPic>
+                       <img src={person.cartoon} alt="team member pic" onMouseOver={e => (e.currentTarget.src =
+                            person.img)} onMouseOut={e => (e.currentTarget.src = person.cartoon)} />
+                   </BioPic>
+               </Col>
+            </Row>;
+        }
+        );
+    }
+
+
+    public render(): any {
+        return (
+            
+            <Background>
+                <Header>
+                    <div className="logo">
+                        <span>O</span>
+                        <span>u</span>
+                        <span>r</span>
+                        <span>_</span>
+                        <span>T</span>
+                        <span>e</span>
+                        <span>a</span>
+                        <span>m</span>
+                        <span className="blink">|</span>
+                    </div>
+                </Header>
+                <Grid>
+
+                    {this.teamMemberList()}
```

Comment:
could we focus a bit on responsiveness for this page? The header, bio's, and pics don't translate well on mobile at the moment. I think just stacking the elements for the bio's/pics should be good enough.